src/fmu/dataio/dataio.py

```diff
@@ -519,6 +552,13 @@ def _show_deprecations_or_notimplemented(self) -> None:
                 PendingDeprecationWarning,
             )
 
+        if self.access_ssdl:
```

Comment:
This is the deprecation warning intended for users who still use the `access_ssdl` argument